src/system/theme/index.js

```diff
@@ -492,7 +523,7 @@ export default {
 			fontFamily: 'body',
 			lineHeight: 'body',
 			fontWeight: 'body',
-			fontSize: 2,
+			fontSize: '16px',
```

Comment:
@djalmaaraujo I see some visual differences if I compare trunk with this branch

Let's thake the accordion
**Trunk**
https://vip-design-system-components.netlify.app/?path=/story/accordion--default
![CleanShot 2024-04-18 at 16 45 28@2x](https://github.com/Automattic/vip-design-system/assets/668251/b4978a70-a670-4761-bf1a-086f8114d90b)


**This branch** 
https://deploy-preview-374--vip-design-system-components.netlify.app/?path=/story/accordion--default
![CleanShot 2024-04-18 at 16 45 41@2x](https://github.com/Automattic/vip-design-system/assets/668251/70b391dc-5c90-4017-b06f-e1302c6a2f66)

Also, links and buttons have bigger fonts. Is this intentional?